components/n-ui/ads/index.js

```diff
@@ -59,13 +60,28 @@ function onAdsComplete (flags, event) {
 	}
 }
 
+//Permutive Trial - config options for o-permutive component.
+const oPermConf = {
+	'appInfo' : {
+		'appName' : 'article',
+		'contentId' : '5cfae92e-6cc5-11e9-80c7-60ee53e6681d'
+	},
+	'publicApiKeys' : {
+		'id' : 'e1c3fd73-dd41-4abd-b80b-4278d52bf7aa',
+		'key' : 'b2b3b748-e1f6-4bd5-b2f2-26debc8075a3'
+	},
+	'adsApi' : {
```

Comment:
Are you not missing the /context endpoint for stream pages?

Reply:
yes, however this is intentional at this stage. The trial is initially just checking that data gets passed through end-to-end to Permutive in the correct way. So just the metadata is just for articles at this stage.

Reply:
Cool cool cool 👍